packages/player/src/Thumbnail.tsx

```diff
@@ -0,0 +1,63 @@
+import {useState} from 'react'
+import type { CSSProperties} from 'react';
+import {useMemo, Suspense} from 'react'
+import type {TComposition, TimelineContextValue} from "remotion";
+import {random, Internals} from "remotion";
+
+export const Thumbnail:React.FC<{
+    composition: TComposition<unknown>;
+    targetHeight: number;
+    targetWidth: number;
+    frameToDisplay: number;
+    style?: CSSProperties
+}> = ({
```

Comment:
Ideally the API is consistent with the one of the `<Player>`.
The redundancy that is in this component is totally fine though, no need to create crazy abstractions 👍